### DISCUSSION TITLE
Miele washingmachine: 90 degrees spin cycle

### DISCUSSION THREAD

**Discussion by haaino** [astrandb/miele#265] [Q&A] [answered]
Hi,

I have a Miele WDD660 washing machine. Once every washing cycle the washing machines prompts me to do a 90 degree washing cycle for maintenance/ kill germs/ etc....

I'm wondering if HACS components receives this information so that I can use it in HASS? if so, what are the parameters I should monitor?

Best regards,
Haaino

**Comment by astrandb** [comment, marked as answer]
No, Miele does not publish that info in the API

**Comment by haaino** [comment]
Oke. Thank you for your quick answer.

**Comment by aturri** [comment]
Hi, 
you could setup a counter into HA reporting the number of days until next hygiene cycle, with an initial value of 30 days (for example) and minimum value 0. You could setup automations to:
- use `counter.decrement` service at each start of day;
- use `counter.reset` service when completing a program whose temperature is >= 60 degrees;
- setup a notification when the counter goes to 0.

I have many counters like this, such as for descaling washing machine, for deep cleaning machine, for filter cleaning, etc.

**Reply by haaino** [reply]
Thank you for your suggestion!
